core/templates/dev/head/pages/exploration_editor/ExplorationStatesService.js

```diff
@@ -245,7 +302,7 @@ oppia.factory('ExplorationStatesService', [
         return getStatePropertyMemento(stateName, 'answer_groups');
       },
       saveInteractionAnswerGroups: function(stateName, newAnswerGroups) {
-        saveStateProperty(stateName, 'answer_groups', newAnswerGroups);
+        saveStateProperty(stateName, 'answer_groups', newAnswerGroups, true);
```

Comment:
Please double-check your work here and elsewhere -- too many args.

Reply:
Done. (Sorry!)